### ISSUE TITLE
[Bug] Deprecated constants which will be removed in HA Core 2025.1

### ISSUE THREAD

**Issue by firstof9** [travisghansen/hass-pfsense#178]
Example log output:
```
2023-12-28 10:19:03.866 WARNING (MainThread) [homeassistant.components.sensor] STATE_CLASS_MEASUREMENT was used from pfsense, this is a deprecated constant which will be removed in HA Core 2025.1. Use SensorStateClass.MEASUREMENT instead, please create a bug report at https://github.com/travisghansen/hass-pfsense/issues
2023-12-28 10:19:03.868 WARNING (MainThread) [homeassistant.components.sensor] STATE_CLASS_MEASUREMENT was used from pfsense, this is a deprecated constant which will be removed in HA Core 2025.1. Use SensorStateClass.MEASUREMENT instead, please create a bug report at https://github.com/travisghansen/hass-pfsense/issues
2023-12-28 10:19:03.870 WARNING (MainThread) [homeassistant.const] DATA_BYTES was used from pfsense, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfInformation.BYTES instead, please create a bug report at https://github.com/travisghansen/hass-pfsense/issues
2023-12-28 10:19:03.871 WARNING (MainThread) [homeassistant.const] FREQUENCY_HERTZ was used from pfsense, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfFrequency.HERTZ instead, please create a bug report at https://github.com/travisghansen/hass-pfsense/issues
2023-12-28 10:19:03.873 WARNING (MainThread) [homeassistant.const] TEMP_CELSIUS was used from pfsense, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please create a bug report at https://github.com/travisghansen/hass-pfsense/issues
2023-12-28 10:19:03.876 WARNING (MainThread) [homeassistant.const] TIME_SECONDS was used from pfsense, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTime.SECONDS instead, please create a bug report at https://github.com/travisghansen/hass-pfsense/issues
```

Should be a pretty simple fix.

**Comment by epenet** [comment]
This issue should be closed - it has already been resolved in v0.6.3
See https://github.com/travisghansen/hass-pfsense/commit/65e9b6215615962a166b7c3c98da90b507f8cc9b

**Comment by firstof9** [comment]
The PR should have closed it, but I'll do it.

Fixed in #183 and #182
